spec/models/merchant_spec.rb

```diff
@@ -64,7 +64,7 @@
       expect(@merchant_1.customers.distinct).to eq([@customer_1, @customer_2, @customer_3, @customer_4, @customer_5, @customer_6])
     end
 
-    it '.ship_ready_items' do
+    xit '.ship_ready_items' do
```

Comment:
Just leaving a comment so I remember to double check this got reactivated

Reply:
It looks like it didn't but since it's already on main let's leave it like this for now. We should figure out what's breaking during our synchronous work time tonight